client.lua

```diff
@@ -175,7 +156,7 @@ Citizen.CreateThread(function()
     while true do
         Wait(1500)
 
-        local player = GetPlayerPed(-1)
+        local player = PlayerPedId()
```

Comment:
If this isn't registered in the same loop does that not mean that the player variable can't be used in the loop below?

Reply:
PlayerPedId() is more optimised then using GetPlayerPed(-1) that's the reason I changed it. It does still work as iv tested it and yes the player local will work in the loop below